includes/Tracking.php

```diff
@@ -196,6 +196,7 @@ private function get_user_properties(): array {
 			'wpVersion'          => get_bloginfo( 'version' ),
 			'phpVersion'         => PHP_VERSION,
 			'isMultisite'        => (int) is_multisite(),
+			'serverEnvironment'  => wp_get_environment_type(),
```

Comment:
We also need to set this in `packages/tracking/src/enableTracking.js`

```
dimension20: 'serverEnvironment',
```

Reply:
Fixed https://github.com/GoogleForCreators/web-stories-wp/pull/10590/commits/4fdf8036a7bf51ac364029a06373564086687632